package.json

```diff
@@ -83,7 +83,7 @@
     "application-config": "^1.0.1",
     "classnames": "^2.3.1",
     "debounce": "^1.2.0",
-    "deltachat-node": "1.79.3",
```

Comment:
shouldn't this be a relative path if node is moved into the repo?

Reply:
? not sure what you mean here, node was moved into the core repo not into the desktop repo